packages/desktop-client/src/components/reports/CategorySelector.tsx

```diff
@@ -1,20 +1,22 @@
 import React, { Fragment, useState } from 'react';
 
-import {
-  type CategoryEntity,
-  type CategoryGroupEntity,
-} from 'loot-core/src/types/models';
+import { type CategoryGroupEntity } from 'loot-core/src/types/models';
 
+import { CheckAll, UncheckAll } from '../../icons/v2';
 import ViewHide from '../../icons/v2/ViewHide';
 import ViewShow from '../../icons/v2/ViewShow';
 import { type CategoryListProps } from '../autocomplete/CategoryAutocomplete';
 import Button from '../common/Button';
+import Text from '../common/Text';
+import View from '../common/View';
 import { Checkbox } from '../forms';
 
+import GraphButton from './GraphButton';
+
 type CategorySelectorProps = {
   categoryGroups: Array<CategoryGroupEntity>;
   selectedCategories: CategoryListProps['items'];
-  setSelectedCategories: (selectedCategories: CategoryEntity[]) => null;
+  setSelectedCategories: (selectedCategories) => null;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What's going on here? We should be adding types instead of removing them.